Packs/Base/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1

```diff
@@ -695,3 +712,21 @@ function ParseDateRange{
     Date(01/01/21)
     #>
 }
+
+function GetIntegrationContext {
```

Comment:
Not sure why we need this utility function. It looks like something that just creates confusion. We already have $demisto.GetIntegrationContext()

Reply:
As discussed, since optional parameters are not supported in class methods at the moment (https://github.com/PowerShell/PowerShell/issues/9701), we'll keep the wrapper function for flexibility and functionality and add a tpb.
Marking this as resolved.